src/server/app/models/meeting.js

```diff
@@ -1,14 +1,20 @@
 module.exports = function (sequelize, DataTypes) {
 
   var Meeting = sequelize.define('Meeting', {
-    title: DataTypes.STRING,
-    closeoutTime: DataTypes.DATE,
-    generalLocation: DataTypes.STRING,
-    duration: DataTypes.INTEGER
+    title: { type: DataTypes.STRING, allowNull: false },
+    closeoutTime: { type: DataTypes.DATE, allowNull: false},
+    generalLocation: { type: DataTypes.STRING, defaultValue: "", allowNull: false }, //  general location
+    latitude: { type: DataTypes.INTEGER, defaultValue: null, validate: { min: -90, max: 90 }}, // Location latitude
```

Comment:
Wait a second, why are there longitude and latitudes on a meeting? These should be in the decision.